src/quo2/components/avatars/group_avatar.cljs

```diff
@@ -21,6 +21,6 @@
                 :align-items      :center
                 :justify-content  :center
                 :border-radius    (/ container-size 2)
-                :background-color (colors/custom-color-by-theme color 50 60)}
+                :background-color color}
```

Comment:
Please don't remove this line, only comment it out.
Also will be better to add a todo, because this is only temporary, we eventually need to support custom colors.